src/sim65/main.c

```diff
@@ -77,21 +85,27 @@ static const unsigned char HeaderVersion = 2;
 /*                                   Code                                    */
 /*****************************************************************************/
 
-
+#ifndef _WIN32
+void Exit() {
+    tcsetattr(STDIN_FILENO, TCSANOW, &OldTermAttrs);
+}
+#endif
 
 static void Usage (void)
 {
     printf ("Usage: %s [options] file [arguments]\n"
             "Short options:\n"
             "  -h\t\t\tHelp (this text)\n"
             "  -c\t\t\tPrint amount of executed CPU cycles\n"
+            "  -t\t\t\tDisable term echo and buffering (*Nix only)\n"
```

Comment:
Maybe the output of the two lines should be optional?

Reply:
Yeap, makes sense ! Done